### DISCUSSION TITLE
Smarthome Light control panel (Philips Hue)

### DISCUSSION THREAD

**Discussion by obranies** [Wendelpunkt/ToypadChallenge#19] [Ideas]
I am thinking about using the Toypad as a convenient light control switch with extra functionality. Different tags (with different physical "MOCs" on them) (Reference: MOC in Lego Language stands for My Own Creation, which is basically anything you create yourself using Lego bricks by not following an official guide). For example, a Mini Figure with red clothing would turn the lights to a red hue, a Lego lamppost would toggle the living room light, and so on.

As I own several Philips Hue lights and accessories, I will focus on leveraging their developer program and SDK.

**Comment by tomwendel** [comment]
Yeah! Love this idea. had the same for controlling my light themes here in the office. In my case I would love to have a connection to the Tuya cloud. But this may be another plugin...
